lecture_1/lecture_1_passed/passed.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/sethgrid/pester"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"time"
```

Comment:
Minor thing but it's usually a good practice to sort imports by first listing the standard library imports, then an empty line and then third-party package imports.

```suggestion
	"encoding/json"
	"fmt"
	"io/ioutil"
	"log"
	"os"
	"strings"
	"time"
	
	"github.com/pkg/errors"
	"github.com/sethgrid/pester"
```